src/tasks/sep/047-soneium-add-game-type/config.toml

```diff
@@ -0,0 +1,41 @@
+templateName = "AddGameTypeTemplate"
+
+l2chains = [    
+    {name = "Soneium Testnet Minato", chainId = 1946},
+]
+
+[[configs]]
+# The below addresses can be found in the Superchain Registry https://github.com/ethereum-optimism/superchain-registry/blob/851f7bb60edc53373fc961d237c7175aa130fba3/superchain/extra/addresses/addresses.json#L402
+chainId = 1946
+saltMixer = "this is a salt mixer"
+systemConfig = "0x4Ca9608Fef202216bc21D543798ec854539bAAd3"
+proxyAdmin = "0xff9d236641962Cebf9DBFb54E7b8e91F99f10Db0"
```

Comment:
do these need to be specified?

Reply:
Okay I tried to remove it and the simulation failed. It matches up to what's in the registry, so you can ignore this comment.